_posts/2017-10-05-progress-report-q3-17.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Progress Report Q3 2017"
+author: pbalcer
+layout: post
+identifier: progress-report-q3-2017
+---
+
+The last quarter was rather... peaceful. But nevertheless there were a few
+noteworthy things.
+
+### FreeBSD & ARM
+
+We always asserted that our library is multi-platform and hardware agnostic...
+as long as your platform is a recent distribution of Linux (or Windows)
+on x86 hardware :)
+
+Two things happened that intend to change the current status quo:
+
+- There's an active and ongoing effort of porting the linux-specific
+parts of the library to FreeBSD, mostly led by [@gaweinbergi](https://github.com/gaweinbergi).
+Our code is naturally very linux centric due to being developed
+alongside the kernel support for NVDIMMs, and there are quite a few
+hurdles to overcome, but everything seems to be going along quite nicely.
+- Patch that ports the low-level library (libpmem) to ARM has been
+submitted to our repository [@vvenkates27](https://github.com/vvenkates27).
+This is also a place where there's a lot of platform specific code due to
+the novelty of the topic and the fact that the library was implemented
+together with the programming model for x86.
```

Comment:
I'd rephrase. This sentence suggests the programming model was designed for x86 and it wasn't. The SNIA doc doesn't mention any specific arch AFAIK.